plugins/inputs/opcua/opcua_client.go

```diff
@@ -392,16 +441,15 @@ func (o *OpcUA) Gather(acc telegraf.Accumulator) error {
 		return err
 	}
 
-	for i, n := range o.NodeList {
+	for i, n := range o.nodes {
 		fields := make(map[string]interface{})
 		tags := map[string]string{
-			"name": n.Name,
-			"id":   BuildNodeID(n),
+			"id": n.idStr,
 		}
 
-		fields[o.NodeData[i].TagName] = o.NodeData[i].Value
-		fields["Quality"] = strings.TrimSpace(fmt.Sprint(o.NodeData[i].Quality))
-		acc.AddFields(o.Name, fields, tags)
+		fields[o.nodeData[i].TagName] = o.nodeData[i].Value
+		fields["quality"] = strings.TrimSpace(fmt.Sprint(o.nodeData[i].Quality))
```

Comment:
For case-sensitive downstream applications (such as InfluxDB IIRC) this might be a problem. First it will generate new series AFAIK as field names are used for hashing **and** it might break queries further down...
Should we really risk this breakage?

Reply:
Telegraf fields are usually lower case.  I wasn't worried about backward compatibility when I changed this, but you have valid concerns.  reverted.